src/daft-functions/src/lib.rs

```diff
@@ -55,6 +55,8 @@ impl FunctionModule for HashFunctions {
     fn register(parent: &mut daft_dsl::functions::FunctionRegistry) {
         parent.add_fn(HashFunction);
         parent.add_fn(MinHashFunction);
+        // Register "hash" as an alias for "hash_with"
+        parent.add_alias("hash", "hash_with");
```

Comment:
Same comment. By keeping the function name as "hash" you don't need to alias it. The only thing we are changing is which rust function were' calling on the arrays because we want to pass an additional argument but in a backwards compatible way to reduce this diff.